### PR TITLE
DAOS-12499 tools: improve error logging of dfs fs get-attr

### PR DESCRIPTION
**Pull request by mchaarawi**
passing pool and container explicitly (not with path) should require the dfs-path argument. This improve the error output to something more user friendly.

Skip-func-test: true

Required-githooks: true

Signed-off-by: Mohamad Chaarawi <mohamad.chaarawi@intel.com>

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

**Review by daltonbohning — APPROVED** [review]
Thanks!

**Review by knard-intel — APPROVED** [review]
LGTM

**Comment by mjmac** [comment]
FYI, I also fixed this in the (much-larger) PR to clean up a bunch of this sort of thing: https://github.com/daos-stack/daos/pull/11317/files#diff-2efa7d20782ffde162f2031491a7d93360186681095c4ef4ea53b1960ebabfb7 ... That's just waiting for a second review. It looks like the two approaches would conflict slightly, so if you don't mind, I'd rather get my version landed.

**Comment by mchaarawi** [comment]
> FYI, I also fixed this in the (much-larger) PR to clean up a bunch of this sort of thing: https://github.com/daos-stack/daos/pull/11317/files#diff-2efa7d20782ffde162f2031491a7d93360186681095c4ef4ea53b1960ebabfb7 ... That's just waiting for a second review. It looks like the two approaches would conflict slightly, so if you don't mind, I'd rather get my version landed.

i didn't see that you updated the help message in your PR. in any case it is failing CI? if you are going to re-push, then please force land this one and rebase with it.

**Review by johannlombardi — APPROVED** [review]

**Comment by mchaarawi** [comment]
> i didn't see that you updated the help message in your PR. in any case it is failing CI? if you are going to re-push, then please force land this one and rebase with it.

oops Johann landed it.. sorry

**Comment by mjmac** [comment]
> > i didn't see that you updated the help message in your PR. in any case it is failing CI? if you are going to re-push, then please force land this one and rebase with it.
> 
> oops Johann landed it.. sorry

No worries, looks like it merged cleanly after all.

**Pull request merged**
